vector/src/main/res/values/strings.xml

```diff
@@ -3228,6 +3228,6 @@
     <string name="device_manager_unverified_sessions_title">Unverified sessions</string>
     <string name="device_manager_unverified_sessions_description">Verify or sign out from unverified sessions.</string>
     <string name="device_manager_inactive_sessions_title">Inactive sessions</string>
-    <string name="device_manager_inactive_sessions_description">Consider signing out from old sessions (90 days or older) you don’t use anymore.</string>
+    <string name="device_manager_inactive_sessions_description">Consider signing out from old sessions (%1$d days or more) that you don’t use anymore.</string>
```

Comment:
Please also use a `plurals` here. Thanks!

Reply:
Fixed in [5791a4d](https://github.com/vector-im/element-android/pull/6965/commits/5791a4d3cdcc7ca096f5ccdc4e2d650027919c31)